src/app/modules/navbar/navbar.component.scss

```diff
@@ -196,6 +196,10 @@
         display: none;
       }
     }
+
+    .nav-btns:first-child {
+      display: block;
+    }
```

Comment:
Not clear why this is needed

Reply:
We have to display `donate button` for all screen sizes.
Earlier all the nav-btns for mobile screen were getting hidden. Since `donate button` is the first child I added `display: block` to that.

**UPDATE**: Since first-child is not a deterministic way of styling, I have now given `donate-btn` class to that particular btn.